ioctl/cmd/account/accountinfo.go

```diff
@@ -71,9 +72,13 @@ func info(arg string) error {
 	if !ok {
 		return output.NewError(output.ConvertError, "", err)
 	}
-
+	ethAddr, err := address.FromString(addr)
+	if err != nil {
+		return output.NewError(output.ConvertError, "", err)
+	}
```

Comment:
can you also add it to `ioctl createadd`?

Reply:
`ioctl account createadd`  is written to file, I don't think it makes much sense.
```
wallet: /Users/millken/.config/ioctl/default
endpoint: api.iotex.one:80
secureConnect: false
aliases:
  mhs: io1tyc2yt68qx7hmxl2rhssm99jxnhrccz9sneh08
  mhs2: io19sdfxkwegeaenvxk2kjqf98al52gm56wa2eqks
  test33: io1xk004px3ljrh0yawypthmjym7pvhndh4l930wk
  yqr: io1cl6rl2ev5dfa988qmgzg2x4hfazmp9vn2g66ng
defaultAccount:
  addressOrAlias: ""
explorer: iotexscan
language: English
nsv2height: 5165641
analyserEndpoint: https://iotex-analyser-api-mainnet.chainanalytics.org
```